WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -797,6 +811,65 @@ extension ProductSelectorViewModel {
     }
 }
 
+// MARK: - Tracking
+private extension ProductSelectorViewModel {
+    func trackConfirmButtonTapped(with productsCount: Int) {
+        let trackingSources = Array(productIDTrackingSources.values.map { $0.rawValue })
```

Comment:
I think this Array cast is duplicated:

```suggestion
        let trackingSources = productIDTrackingSources.values.map { $0.rawValue }
```

Reply:
Indeed, fixed in https://github.com/woocommerce/woocommerce-ios/pull/9586